epub33/core/index.html

```diff
@@ -1470,15 +1464,16 @@ <h3>Data URLs</h3>
 					directly into a URL string. The advantage of this scheme is that it allows [=EPUB creators=] to
 					embed a resource within another, avoiding the need for an external file.</p>
 
-				<p>EPUB creators MAY use data URLs in [=EPUB publications=] provided their use does not result in a
-					[=top-level content document=] or [=top-level browsing context=] [[html]]. This restriction applies
-					to data URLs used in the following scenarios:</p>
+				<p>A consequence of embedding is that data URLs are not considered their own unique [=publication
+					resources=] (i.e., they are part of their containing publication resource). As a result, EPUB
```

Comment:
Purely editorial issue: a data URL is, conceptually, a URL which also _defines_ some data. So the question whether it is a publication resource or not does not apply to the data URL itself, but to the data it defines.

But this is really a nuance and it may be simpler to ignore it here.

Reply:
We could say: "A consequence of embedding is that the data in a data URL is not considered a unique publication resource (i.e., it is part of the containing publication resource)."

I don't think that's particularly hard to parse, if that's what you're getting after?

Reply:
Yep, that sounds good to me